SchoolHousing/SchoolHousing/Pages/Index.cshtml

```diff
@@ -7,14 +7,13 @@
 }
 
 
-<h1 class="display-4">Affordable Houses near good elementary schools!</h1>
+<h1 class="display-4"><center>List of Affordable Houses near Good Elementary Schools!</center></h1>
```

Comment:
Avoid use of the center tag.  That is not XHTML 1.0 Strict compliant.  Use CSS instead.